docs/source/components.rst

```diff
@@ -0,0 +1,332 @@
+==========
+Components
+==========
+
+This document describes all components or "steps" the new Django serializers
```

Comment:
Docs should be present tense (even if the code does not exist yet), and for now let's not explicitly say these will be Django's new serializers